libs/saksbehandling-common/src/main/kotlin/behandling/BoddEllerArbeidetUtlandet.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.etterlatte.libs.common.behandling
+
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+
+data class BoddEllerArbeidetUtlandet(
```

Comment:
Kjenner vi har behov for ein _etterlatte-behandling-model_ snart, men denne PR-en er jo meir enn stor nok frå før 😅

Reply:
Kommer muligens noen spørsmål til - og de vil kanskje bety at vi må endre layout/konsept litt på søknadsoversikt. Føler at slike endringer er kanskje bedre der enn her.